[2022-08-03 02:14:17,231] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-08-03T02:14:13.464284+00:00 [queued]>
[2022-08-03 02:14:17,271] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-08-03T02:14:13.464284+00:00 [queued]>
[2022-08-03 02:14:17,272] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 02:14:17,272] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-08-03 02:14:17,273] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 02:14:17,313] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-08-03 02:14:13.464284+00:00
[2022-08-03 02:14:17,338] {standard_task_runner.py:52} INFO - Started process 118 to run task
[2022-08-03 02:14:17,384] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'download_dataset_task', 'manual__2022-08-03T02:14:13.464284+00:00', '--job-id', '104', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmp09gx_cd2', '--error-file', '/tmp/tmpp7ukmn46']
[2022-08-03 02:14:17,387] {standard_task_runner.py:77} INFO - Job 104: Subtask download_dataset_task
[2022-08-03 02:14:17,840] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-08-03T02:14:13.464284+00:00 [running]> on host de687df7354f
[2022-08-03 02:14:18,113] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-08-03T02:14:13.464284+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-03T02:14:13.464284+00:00
[2022-08-03 02:14:18,122] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-03 02:14:18,123] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://storage.googleapis.com/dtc_data_lake_de-bootcamp-357803/covid-cases.csv > /opt/***/covid-cases.csv']
[2022-08-03 02:14:18,148] {subprocess.py:85} INFO - Output:
[2022-08-03 02:14:19,482] {subprocess.py:93} INFO - Command exited with return code 0
[2022-08-03 02:14:19,528] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=download_dataset_task, execution_date=20220803T021413, start_date=20220803T021417, end_date=20220803T021419
[2022-08-03 02:14:19,603] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-08-03 02:14:19,671] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
